### ISSUE TITLE
Live Docs stopped updating

### ISSUE THREAD

**Issue by ssfrr** [fonsp/Pluto.jl#2346]
I'm looking through the example notebooks in a local Pluto.jl instance, and the Live Docs view stopped updating, even after opening a different notebook. They started working again after re-loading the page, however. I looked in the javascript console and see the following message:

```
Ignoring action set_doc_query 
Object { args: (1) […] }
​args: Array [ "bind" ]
​​0: "bind"
​​length: 1
​​<prototype>: Array []
​<prototype>: Object { … }
​​__defineGetter__: function __defineGetter__()
​​__defineSetter__: function __defineSetter__()
​​__lookupGetter__: function __lookupGetter__()
​​__lookupSetter__: function __lookupSetter__()
​​__proto__: 
​​constructor: function Object()
​​hasOwnProperty: function hasOwnProperty()
​​isPrototypeOf: function isPrototypeOf()
​​propertyIsEnumerable: function propertyIsEnumerable()
​​toLocaleString: function toLocaleString()
​​toString: function toString()
​​valueOf: function valueOf()
​​<get __proto__()>: function __proto__()
​​<set __proto__()>: function __proto__()
editor.ed426e4a.js:673:582
```

Here's the package manifest:
```
(@v1.8) pkg> st -m
Status `~/.julia/environments/v1.8/Manifest.toml`
  [d1d4a3ce] BitFlags v0.1.5
⌃ [da1fd8a2] CodeTracking v1.1.0
  [944b1d66] CodecZlib v0.7.0
  [8f4d0f93] Conda v1.7.0
  [5218b696] Configurations v0.17.4
  [9a962f9c] DataAPI v1.12.0
  [e2d170a0] DataValueInterfaces v1.0.0
  [55351af7] ExproniconLite v0.7.1
  [fb4132e2] FuzzyCompletions v0.5.1
  [cd3eb016] HTTP v1.5.1
  [ac1192a8] HypertextLiteral v0.9.4
  [7073ff75] IJulia v1.23.3
  [83e8ac13] IniFile v0.5.1
  [82899510] IteratorInterfaceExtensions v1.0.0
  [692b3bcd] JLLWrappers v1.4.1
  [682c06a0] JSON v0.21.3
  [aa1ae85d] JuliaInterpreter v0.9.15
  [0e77f7df] LazilyInitializedFields v1.2.0
  [e6f89c97] LoggingExtras v0.4.9
  [6f1432cf] LoweredCodeUtils v2.2.2
  [6c6e2e6c] MIMEs v0.1.4
⌃ [739be429] MbedTLS v1.1.6
  [99f44e22] MsgPack v1.1.0
  [4d8831e6] OpenSSL v1.3.0
  [bac558e1] OrderedCollections v1.4.1
⌃ [69de0a69] Parsers v2.4.0
  [c3e4b0f8] Pluto v0.19.14
  [91cefc8d] PrecompileSignatures v3.0.3
  [21216c6a] Preferences v1.3.0
  [2792f1a3] RegistryInstances v0.1.0
⌅ [05181044] RelocatableFolders v0.3.0
  [ae029012] Requires v1.3.0
  [295af30f] Revise v3.4.0
  [6c6a2e73] Scratch v1.1.1
  [777ac1f9] SimpleBufferStream v1.1.0
  [b85f4697] SoftGlobalScope v1.1.0
  [3783bdb8] TableTraits v1.0.1
  [bd369af6] Tables v1.10.0
  [3bb67fe8] TranscodingStreams v0.9.9
  [410a4b4d] Tricks v0.1.6
  [5c2747f8] URIs v1.4.0
  [81def892] VersionParsing v1.3.0
⌃ [c2297ded] ZMQ v1.2.1
  [458c3c95] OpenSSL_jll v1.1.17+0
  [8f1865be] ZeroMQ_jll v4.3.4+0
  [a9144af2] libsodium_jll v1.0.20+0
  [0dad84c5] ArgTools v1.1.1
  [56f22d72] Artifacts
  [2a0f44e3] Base64
  [ade2ca70] Dates
  [8ba89e20] Distributed
  [f43a241f] Downloads v1.6.0
  [7b1f6079] FileWatching
  [b77e0a4c] InteractiveUtils
  [b27032c2] LibCURL v0.6.3
  [76f85450] LibGit2
  [8f399da3] Libdl
  [37e2e46d] LinearAlgebra
  [56ddb016] Logging
  [d6f4376e] Markdown
  [a63ad114] Mmap
  [ca575930] NetworkOptions v1.2.0
  [44cfe95a] Pkg v1.8.0
  [de0858da] Printf
  [3fa0cd96] REPL
  [9a3f8284] Random
  [ea8e919c] SHA v0.7.0
  [9e88b42a] Serialization
  [6462fe0b] Sockets
  [fa267f1f] TOML v1.0.0
  [a4e569a6] Tar v1.10.0
  [8dfed614] Test
  [cf7118a7] UUIDs
  [4ec0a83e] Unicode
  [e66e0078] CompilerSupportLibraries_jll v0.5.2+0
  [deac9b47] LibCURL_jll v7.84.0+0
  [29816b5a] LibSSH2_jll v1.10.2+0
  [c8ffd9c3] MbedTLS_jll v2.28.0+0
  [14a3606d] MozillaCACerts_jll v2022.2.1
  [4536629a] OpenBLAS_jll v0.3.20+0
  [83775a58] Zlib_jll v1.2.12+3
  [8e850b90] libblastrampoline_jll v5.1.1+0
  [8e850ede] nghttp2_jll v1.48.0+0
  [3f19e933] p7zip_jll v17.4.0+0
```

**Comment by fonsp** [comment]
Thanks for the issue! Are you able to reproduce the issue? Could you send us a video recording of the problem? 

Instructions are [here](https://github.com/fonsp/Pluto.jl/blob/v0.19.14/.github/ISSUE_TEMPLATE/bug_report.md)

**Comment by fonsp** [comment]
It looks like this was caused by the same reason as #2215 , which was fixed by #2261, so we need a similar fix for this issue.

**Comment by fonsp** [comment]
I can't reproduce this issue. It could be that the backend was still booting up a notebook process while the first request was sent, causing the console log, and subsequent doc searches did work, but they were still compiling the julia code, or waiting for the notebook to finish running, making it appear broken.
